kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -769,6 +834,17 @@ def parse(s: str) -> List[T]:
     view_kcfg_args.add_argument('spec_file', type=file_path, help='Path to spec file.')
     view_kcfg_args.add_argument('--claim', type=str, dest='claim_label', help='Claim identifier to load CFG for.')
 
+    show_kcfg_args = command_parser.add_parser(
+        'show-kcfg',
+        help='Display tree show of CFG',
+        parents=[shared_args, k_args, kcfg_show_args, display_args],
+    )
+    show_kcfg_args.add_argument(
+        'save_directory', type=dir_path, help='Path to where CFGs are stored (--save-directory option to prove).'
+    )
+    show_kcfg_args.add_argument('spec_file', type=file_path, help='Path to spec file.')
+    show_kcfg_args.add_argument('--claim', type=str, dest='claim_label', help='Claim identifier to load CFG for.')
```

Comment:
These three arguments for 'view_kcfg_args' and 'show_kcfg_args' are the exact same. Should we pull them out into some shared 'kcfg_args'?

Reply:
I factored out `spec_args`, which I used for `prove_args`, `show_kcfg_args`, and `view_kcfg_args`. Now they all take exactly the same arguments.

Reply:
Likely this `spec_args` should be factored out to upstream.

Reply:
https://github.com/runtimeverification/pyk/issues/260